lib/content.ex

```diff
@@ -24,17 +24,17 @@ defmodule Content do
   """
   def call(conn, options) do
     if get_accept_header(conn) =~ "json" do
-      # for json requests return the conn unmodified:
-      # if we need options.json_plugs in the future, we can add it.
+      #  for json requests return the conn unmodified:
+      #  if we need options.json_plugs in the future, we can add it.
       conn
     else
-      # if accept header not "json" then assume "html" (the default)
+      #  if accept header not "json" then assume "html" (the default)
       # invoke each function in the list of html_plugs
       # and pass the conn as accumulator through each iteration
-      # return the conn with all html_plugs applied to it.
+      #  return the conn with all html_plugs applied to it.
```

Comment:
this whitespace appears to be added randomly ... 🤔

Reply:
I think it might be the formatter, I'll run it again to check